dev/make-distribution.sh

```diff
@@ -201,6 +201,12 @@ fi
 # Copy data files
 cp -r "$SPARK_HOME/data" "$DISTDIR"
 
+# Make pip package
+echo "Building python distribution package"
+cd $SPARK_HOME/python
+python setup.py sdist
```

Comment:
Can we be sure that the machine building distribution always has python installed? If the users doesn't think about to use pyspark, original procedure of making distribution doesn't ask them to do anything with pyspark. But with this change, it seems making assumption all users want to have python distribution.